content/docs/user-guide/basic-concepts.md

```diff
@@ -0,0 +1,34 @@
+# Basic Concepts of DVC
```

Comment:
so, how do we connect this with glossary?

Reply:
I just extracted some info from What is DVC? to this new basic concepts that will kickstart #72 but it's not added to the navigation, so it's hidden for now. I know this doc needs lots of work and figure out the glossary connection but I'm leaving all that for a separate PR.

Reply:
I would say we don't have that much useful info here to keep it

Reply:
There's a good amount now. And more coming soon... ⌛

Reply:
Extracted this to #1655

Reply:
I still don't see much new compared to the glossary in this format. For now it just means two similar places to keep updated for us.

 Also it's strange a bit to have advanced concepts in basic concepts. Also, split itself is quite artificial.

Reply:
#1655 is where we'll merge it with the glossary. This PR is just about dissolving the What is DVC section. Should I hide the Basic Concepts guide from the nav for now?

Reply:
Basic vs advance concepts will also be resolved in #1655.

Reply:
I just don't see any value in the page as it is done now. We would need to spend time reviewing it, etc.

Reply:
I agree, that's what I'm working on now/next. It's an old need: #550 — we should resolve it once and for all 🙂

Reply:
To clarify. My take on #550 is that it's way more than a single page like this. Good example of a basic concept is cache, or run cache - they alone can take a proper page I guess.

Reply:
Noted. We can break it up in #1655. Should we move the discussion there and get this one merged? Or is having the guide here blocking the approval? Because I can just remove it and any links to it for now, and re-introduce it in that other PR. Please lmk

Reply:
I guess, let's remove it for now and simplify the PR

Reply:
so, are we removing this?

Reply:
Oh I read that wrong. I though you said "let's leave it for now". OK, removed now.